renderer/pages/config.js

```diff
@@ -5,6 +5,8 @@ import {ConfigContainer} from '../containers';
 import Config from '../components/config';
 import WindowHeader from '../components/window-header';
 
+import '../utils/sentry';
```

Comment:
I wonder if we can add it to _app.js, and that way dont need to add it to every page 🤔 See https://github.com/zeit/next.js#custom-app

Reply:
https://github.com/zeit/next.js/blob/canary/examples/with-sentry/pages/_app.js